circle.yml

```diff
@@ -58,20 +58,29 @@ jobs:
       - checkout
       - run:
           name: Generate Maven credentials
-          shell: /bin/bash -euo pipefail
           command: |
-            aws s3 cp s3://mapbox/android/signing-credentials/secring.gpg secring.gpg
-            echo "NEXUS_USERNAME=$PUBLISH_NEXUS_USERNAME
-            NEXUS_PASSWORD=$PUBLISH_NEXUS_PASSWORD
-            signing.keyId=$SIGNING_KEYID
-            signing.password=$SIGNING_PASSWORD
-            signing.secretKeyRingFile=../secring.gpg" >> gradle.properties
+            if [ -n "${BINTRAY_USER}" ]; then
+              echo "BINTRAY_USER=$BINTRAY_USER
+              BINTRAY_API_KEY=$BINTRAY_API_KEY
+              GPG_PASSPHRASE=$GPG_PASSPHRASE"
+            fi
+      - run:
+          name: Update version name
+          command: |
+            if [[ $CIRCLE_TAG == v* ]]; then
+              sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${CIRCLE_TAG:9}/" gradle.properties
+            fi          
       - run:
           name: Build Java libraries
           command: make build-release
       - deploy:
-          name: Publish Java Services To Maven Central
+          name: Publish Java libraries to Bintray
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            make publish ;
+            if [[ $CIRCLE_BRANCH == master ]] || [[ $CIRCLE_TAG == android-v* ]]; then
```

Comment:
`[[ $CIRCLE_TAG == android-v* ]]` should be `[[ $CIRCLE_TAG == v* ]]`

Reply:
Fixed